File: HPWorkpath_v1.6.3/Samples/ExampleExtensions/source/GoogleSigninSample/src/com/hp/workpath/sample/googlesigninsample/model/ClientInfo.java
// Copyright 2018 HP Development Company, L.P.
// SPDX-License-Identifier: MIT
package com.hp.workpath.sample.googlesigninsample.model;

import com.google.gson.annotations.SerializedName;

public class ClientInfo {

    @SerializedName("client_id")
    String clientId;

    @SerializedName("client_secret")
    String clientSecret;

    @SerializedName("redirect_uri")
    String redirectUri;

    @SerializedName("grant_type")
    String grantType;

    @SerializedName("code_challenge_method")
    String codeChallengeMethod;

    @SerializedName("auth_uri")
    String authUri;

    @SerializedName("token_uri")
    String tokenUri;

    String codeChallenge;

    String codeVerifier;

    public String getClientId() {
        return clientId;
    }

    public String getClientSecret() {
        return clientSecret;
    }

    public String getGrantType() {
        return grantType;
    }

    public String getRedirectUri() {
        return redirectUri;
    }

    public String getCodeChallengeMethod() {
        return codeChallengeMethod;
    }

    public String getAuthUri() {
        return authUri;
    }

    public String getTokenUri() {
        return tokenUri;
    }

    public String getCodeChallenge() {
        return codeChallenge;
    }

    public void setCodeChallenge(String codeChallenge) {
        this.codeChallenge = codeChallenge;
    }

    public String getCodeVerifier() {
        return codeVerifier;
    }

    public void setCodeVerifier(String codeVerifier) {
        this.codeVerifier = codeVerifier;
    }
}

File: HPWorkpath_v1.6.3/Samples_Kotlin/ExampleExtensions/source/GoogleSigninSample/src/com/hp/workpath/sample/googlesigninsample/model/ClientInfo.kt
// Copyright 2018 HP Development Company, L.P.
// SPDX-License-Identifier: MIT
package com.hp.workpath.sample.googlesigninsample.model

import com.google.gson.annotations.SerializedName

class ClientInfo {
    @SerializedName("client_id")
    var clientId: String? = null

    @SerializedName("client_secret")
    var clientSecret: String? = null

    @SerializedName("redirect_uri")
    var redirectUri: String? = null

    @SerializedName("grant_type")
    var grantType: String? = null

    @SerializedName("code_challenge_method")
    var codeChallengeMethod: String? = null

    @SerializedName("auth_uri")
    var authUri: String? = null

    @SerializedName("token_uri")
    var tokenUri: String? = null
    var codeChallenge: String? = null
    var codeVerifier: String? = null
}